packages/client/src/v2-events/components/forms/FormFieldGenerator/GeneratedInputField.tsx

```diff
@@ -166,6 +168,21 @@ export const GeneratedInputField = React.memo(
         </InputField>
       )
     }
+
+    if (isDateRangeFieldType(field)) {
+      return (
+        <InputField {...field.inputFieldProps}>
+          <DateRangeField.Input
+            {...inputProps}
+            value={field.value}
+            onChange={(val) => {
+              // @ts-ignore
```

Comment:
Whats wrong with the typing here exactly?

Reply:
When it is a complex type, it throws a linter error, just like the address field. Complex type because DateRangePicker can select both an exact date and a date range, just like an address based on urban or rural. Need to add this comment:
```
//@TODO: We need to come up with a general solution for complex types.
```